esp-hal-embassy/CHANGELOG.md

```diff
@@ -92,4 +98,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 0.1.0 - 2024-06-04
 
-[Unreleased]: https://github.com/esp-rs/esp-hal/commits/main/esp-hal-embassy?since=2025-01-15
+[0.7.0]: https://github.com/esp-rs/esp-hal/releases/tag/esp-hal-embassy-v0.7.0
```

Comment:
I don't think we have a `[0.7.0]` link anywhere in this changelog, so why do we have the definition for it?

Reply:
Ah I see L18 should say `[0.7.0] - ...`